modules/disk-uploader/pkg/image/image.go

```diff
@@ -16,7 +17,24 @@ import (
 	tar "kubevirt.io/containerdisks/pkg/build"
 )
 
-func Build(diskPath string) (v1.Image, error) {
+const (
+	defaultInstanceType = "INSTANCETYPE_KUBEVIRT_IO_DEFAULT_INSTANCETYPE"
+	defaultPreference   = "INSTANCETYPE_KUBEVIRT_IO_DEFAULT_PREFERENCE"
+)
+
+func DefaultConfig(labels map[string]string) v1.Config {
+	var env []string
+
+	if value, exists := labels["instancetype.kubevirt.io/default-instancetype"]; exists {
```

Comment:
please use constant for `instancetype.kubevirt.io/default-instancetype`, the same for preference